src/Discord.php

```diff
@@ -36,7 +38,7 @@ public function __construct(
     }
 
     /**
-     * @param Bitwise<\Ragnarok\Fenrir\Enums\Gateway\Intents> $intents
+     * @param Bitwise<Intents> $intents
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)

Reply:
Understood, thanks for clarifying.